boards/arm/stm32f429i_disc1/stm32f429i_disc1.dts

```diff
@@ -235,7 +235,7 @@
 		     &ltdc_hsync_pc6 &ltdc_vsync_pa4>;
 	pinctrl-names = "default";
 	ext-sdram = <&sdram2>;
-	display-controller = <&ili9341>;
+	panel-controller = "ilitek,ili9341";
```

Comment:
What if there are 2 instance of this controller ? How would you be able to define the one to use ?